content/id/docs/reference/glossary/workload.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Beban Kerja
+id: workloads
+date: 2019-02-13
+full_link: /id/docs/concepts/workloads/
+short_description: >
+  Sebuah beban kerja merupakan suatu aplikasi yang berjalan pada Kubernetes.
+
+aka:
+- Workload
+tags:
+- fundamental
+---
+Sebuah beban kerja merupakan suatu aplikasi yang berjalan pada Kubernetes.
+
+<!--more-->
+
+Beragam objek inti yang merepresentasikan tipe atau bagian berbeda dari suatu beban kerja termasuk objek DaemonSet, Deployment, Job, ReplicaSet, dan StatefulSet.
+
+Sebagai contoh, suatu beban kerja dengan sebuah server web dan pangkalan data mungkin menjalankan pangkalan data dalam satu {{< glossary_tooltip term_id="StatefulSet" >}} dan server web dalam satu {{< glossary_tooltip term_id="Deployment" >}}.
```

Comment:
nit!
```suggestion
Sebagai contoh, suatu beban kerja dengan sebuah server jejaring (_web_) dan pangkalan data mungkin menjalankan pangkalan data dalam satu {{< glossary_tooltip term_id="StatefulSet" >}} dan server web dalam satu {{< glossary_tooltip term_id="Deployment" >}}.
```

Reply:
web is valid kbbi too https://kbbi.web.id/web, and has been used, e.g. here https://id.wikipedia.org/wiki/Server_web